.github/workflows/linux_arm64.yml

```diff
@@ -47,20 +48,42 @@ jobs:
           echo "${{ toJSON(steps.smart_ci.outputs.affected_components) }}"
         shell: bash
 
-  Build:
+  Docker:
     needs: Smart_CI
+    runs-on: aks-linux-16-cores-arm-docker-build
+    container:
+      image: openvinogithubactions.azurecr.io/docker_build:0.2
+      volumes:
+        - /mount:/mount
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_20_04_arm64
```

Comment:
will be those images built in parallel? Maybe it make since to split this step on 2 (build image and test image)?

Reply:
The images are not built in parallel, we can optimize it later, but for now I think it's quick enough without extra optimizations (10 mins or so even without cache)